proposals/description/tip_118.ts

```diff
@@ -379,25 +358,44 @@ const tip_118: TemplatedProposalDescription = {
       description: 'Move all agEUR to redeemer'
     },
     {
-      target: 'collateralizationOracle',
+      target: 'angleEuroRedeemer',
       values: '0',
-      method: 'removeDeposits(address[])',
-      arguments: (addresses) => [[addresses.agEurUniswapPCVDeposit, addresses.uniswapLensAgEurUniswapGauge]],
-      description: 'Remove agEUR addresses from CR oracle'
+      method: 'redeemAgEurToDai()',
+      arguments: (addresses) => [],
+      description: 'Redeem all agEUR for DAI and send to DAI PSM'
     },
+
+    // 4. Unset various safe addresses - deprecated PSMs and agEurUniswapPCVDeposit
     {
       target: 'pcvGuardianNew',
       values: '0',
-      method: 'unsetSafeAddress(address)',
-      arguments: (addresses) => [addresses.agEurUniswapPCVDeposit],
-      description: 'Remove agEurUniswapPCVDeposit from safe addresses'
+      method: 'unsetSafeAddresses(address[])',
+      arguments: (addresses) => [
+        [addresses.ethPSM, addresses.lusdPSM, addresses.raiPriceBoundPSM, addresses.agEurUniswapPCVDeposit]
+      ],
+      description:
+        'Unset the ETH, LUSD and RAI PSMs as safe addresses. Unset agEurUniswapPCVDeposit as a safe addresses'
     },
+
+    // Remove various deposits from CR
     {
-      target: 'angleEuroRedeemer',
+      target: 'collateralizationOracle',
       values: '0',
-      method: 'redeemAgEurToDai()',
-      arguments: (addresses) => [],
-      description: 'Redeem all agEUR for DAI and send to DAI PSM'
+      method: 'removeDeposits(address[])',
+      arguments: (addresses) => [
+        [
+          addresses.agEurUniswapPCVDeposit,
+          addresses.uniswapLensAgEurUniswapGauge,
+          addresses.lusdPSM,
+          addresses.ethPSM,
+          addresses.aaveEthPCVDepositWrapper,
+          addresses.voltDepositWrapper
+        ]
```

Comment:
Should we remove these from CR oracle ? They're empty or almost empty : 
- [turboFusePCVDeposit](https://etherscan.io/address/0x2c47Fef515d2C70F2427706999E158533F7cF090) (empty)
- [feiOATimelockWrapper](https://etherscan.io/address/0x7Eb88140af813294aEDce981b6aC08fcd139d408) (empty)
- [rariPool128FeiPCVDepositWrapper](https://etherscan.io/address/0xA62ddde8F799873E6FcdbB3aCBbA75da85D9dcdE) (25k left and configured in the [fuseWithdrawalGuard](https://etherscan.io/address/0xd079ec4b442600a381eac7e95662eb1b313cd113))
- [rariPool22FeiPCVDepositWrapper](https://etherscan.io/address/0xa2BdbCb95d31C85BAE6f0FA42D55F65d609D94eE) (34k left and configured in the [fuseWithdrawalGuard](https://etherscan.io/address/0xd079ec4b442600a381eac7e95662eb1b313cd113))

Reply:
Good catches, yep, removing